lib/dbs/mysql.ts

```diff
@@ -91,3 +91,14 @@ export async function getStoreToken(storeHash: string) {
 export async function deleteStore({ store_hash: storeHash }: SessionProps) {
     await query('DELETE FROM stores WHERE storeHash = ?', storeHash);
 }
+
+export async function hasAppExtensionsScope(storeHash: string): Promise<boolean> {
+    const scopes = await query('SELECT scope FROM stores WHERE storeHash = ?', storeHash);
+    const row = scopes[0];
+
+    if (!row) {
+       return false;
+    }
```

Comment:
🍹 Can probably one-line this `if` statement.
```suggestion
    if (!row) return false;
```